bigtable/docs/snippets_table.py

```diff
@@ -403,5 +412,77 @@ def test_bigtable_table_row():
     table.truncate(timeout=300)
 
 
-if __name__ == "__main__":
+def test_bigtable_add_row_add_row_range_add_row_range_from_keys():
+    row_keys = [
+        b'row_key_1', b'row_key_2', b'row_key_3', b'row_key_4',
+        b'row_key_5', b'row_key_6', b'row_key_7', b'row_key_8',
+        b'row_key_9']
+
+    rows = []
+    for row_key in row_keys:
+        row = Config.TABLE.row(row_key)
+        row.set_cell(COLUMN_FAMILY_ID, COL_NAME1, CELL_VAL1)
+        rows.append(row)
+    Config.TABLE.mutate_rows(rows)
+
+    # [START bigtable_add_row_key]
+    from google.cloud.bigtable import Client
```

Comment:
I don't think we need the Client related code here.  The RowSet code is enough.

Reply:
This follows the format we use for the api docs samples for python, so I think it is fine to leave this in.